app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java

```diff
@@ -303,7 +306,7 @@ public void onFsViewerSelected(String request, File file, final Integer lineNumb
                             @Override
                             public void onFsViewerConfig(FilesystemViewerData.Options dopt) {
                                 dopt.titleText = R.string.todo;
-                                dopt.rootFolder = Environment.getExternalStorageDirectory();
+                                dopt.rootFolder = AppSettings.get().getNotebookDirectory();
```

Comment:
Start quicknote and todo in notebook, not storage root

Reply:
please use context from fragment/activity as far possible. Generally that static Context instance nedes to go at some point (=activity/context leak)